aequilibrae/paths/results/assignment_results.py

```diff
@@ -109,6 +109,7 @@ def prepare(self, graph: Graph, matrix: AequilibraeMatrix) -> None:
         self.crosswalk = np.zeros(graph.graph.shape[0], self.__integer_type)
         self.crosswalk[graph.graph.__supernet_id__.values] = graph.graph.__compressed_id__.values
```

Comment:
Do we use the crosswalk for anything?

Reply:
Yeah, its still used within `all_or_nothing.py` and `linear_approximation.py`
```python
aequilibrae/paths/all_or_nothing.py
80:            self.results.link_loads, self.results.compact_link_loads, self.results.crosswalk, self.results.cores
aequilibrae/paths/linear_approximation.py
389:                aggregate_link_costs(cost, c.graph.compact_cost, c.results.crosswalk)
486:                    aggregate_link_costs(self.congested_time[:], c.graph.compact_skims[:, k], c.results.crosswalk)
aequilibrae/paths/results/assignment_results.py
 42:        self.crosswalk = np.array([])  # crosswalk between compact graph link IDs and actual link IDs
109:        self.crosswalk = np.zeros(graph.graph.shape[0], self.__integer_type)
110:        self.crosswalk[graph.graph.__supernet_id__.values] = graph.graph.__compressed_id__.values
```